program/program/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+
+namespace ConsoleApp
+{
+    class Program
+    {
+        
+        
+                        
+        static void Main()
```

Comment:
it is not the console interface I can interact with, e.g. add questions, vote for something, etc.